client/src/components/PageLogIn/PageLogIn.jsx

```diff
@@ -21,6 +25,7 @@ const PageLogIn = () => {
   }
 
   const onClickButton = () => {
+    dispatch(changeAuthStatusInStore(true));
```

Comment:
You change auth status regardless to whether login was successful or not. I think, you should check, if an error occurred. Though maybe it's worth a separate PR.